spec/openapi.yaml

```diff
@@ -5649,6 +5689,41 @@ components:
         - label
         - schedule
         - report
+      properties:
```

Comment:
@jamie-thomson-sinch think this needs to be reviewed as it looks like `scheduledsummaryreport` is no longer used so this addition is like redundant?

Reply:
<img width="855" alt="Screenshot 2023-10-03 at 2 42 33 pm" src="https://github.com/messagemedia/documentation/assets/116612529/b0bfa612-96ae-4b1d-a22f-a4881cdeee52">
Not sure if that screenshot worked, but it is used by "Scheduled summary report", line 959

Reply:
Would this suggest we do not support `/v2-preview/reporting/summary/scheduled` and thus should remove it from the docs too?

Reply:
No trouble then, looks good 👍